pumpkin/src/entity/player.rs

```diff
@@ -436,6 +436,10 @@ impl Player {
 
         self.living_entity.tick();
 
+        // TODO: maybe find better way then queuing for just sending data
+        let level_time = self.living_entity.entity.world.level_time.lock().await;
+        level_time.send_time(&self.client).await;
```

Comment:
Do this in world not in the Player. Also when looking at vanilla code it does
```java
 if (this.ticks % 20 == 0) {
    this.sendTimeUpdatePackets(serverWorld);
}
```

Reply:
all resources online said it sent the update every tick. I thought it was every second. where does this code come from? is there a link to documentation?